src/commands/overrides/update.ts

```diff
@@ -0,0 +1,95 @@
+import {
+    environmentPrompt,
+    EnvironmentPromptResult,
+    featurePrompt, FeaturePromptResult,
+} from '../../ui/prompts'
+import { Flags } from '@oclif/core'
+import inquirer from '../../ui/autocomplete'
+import { variationPrompt } from '../../ui/prompts/variationPrompts'
+import { UpdateOverrideDto } from '../../api/schemas'
+import { updateOverride } from '../../api/overrides'
+import Base from '../base'
+import { fetchUserProfile } from '../../api/userProfile'
+
+export default class UpdateOverride extends Base {
+    static hidden = false
+    authRequired = true
+    static description = 'Update an override'
```

Comment:
To be consistent with the other descriptions and usages, let's capitalize `Override` here, in the flag descriptions, and the error/success/info messages